test.js

```diff
@@ -105,6 +105,18 @@ function setCouchDBEnv(container, next) {
     // container's port 80 is dynamically mapped to an external port
     var port = _.get(c,
       ['NetworkSettings', 'Ports', '5984/tcp', '0', 'HostPort']);
+
+    var skips = {
+      skip: [
+        'find all limt ten',
+        'find all skip ten limit ten',
+        'find all skip two hundred',
+        'isActual'
+      ],
+    };
+
+    process.env.LOOPBACK_MOCHA_SKIP = JSON.stringify(skips);
```

Comment:
Is it possible that we define the env in `init.js` so people run test with `test.js` or `npm run mocha` can both have the skipped test config?